eva-pipeline/src/main/java/embl/ebi/variation/eva/pipeline/annotation/load/VariantAnnotationMongoItemWriter.java

```diff
@@ -0,0 +1,168 @@
+ *      { "id" : "ENST00000608838", "src" : "ensemblTranscript" },
+ *      { "id" : "ENSG00000178591", "src" : "ensemblGene"
+ */
+public class VariantAnnotationMongoItemWriter extends MongoItemWriter<VariantAnnotation> {
+    private static final Logger logger = LoggerFactory.getLogger(VariantAnnotationMongoItemWriter.class);
+
+    private MongoOperations mongoOperations;
+    private String collection;
+
+    public VariantAnnotationMongoItemWriter(MongoOperations mongoOperations) {
+        this.mongoOperations = mongoOperations;
+    }
+
+    @Override
+    public void setCollection(String collection) {
+        super.setCollection(collection);
+        this.collection = collection;
+    }
+/*    public VariantAnnotationMongoItemWriter() {
+        this.template = mongoTemplate();
+    }
+
+    public MongoTemplate mongoTemplate() {
+        MongoTemplate mongoTemplate;
+        try {
+            //mongoTemplate = new MongoTemplate(new MongoClient(), pipelineOptions.getString(VariantStorageManager.DB_NAME));
+            mongoTemplate = new MongoTemplate(new MongoClient(), "variants");
+        } catch (UnknownHostException e) {
+            throw new RuntimeException("Unable to initialize MongoDB", e);
+        }
+        return mongoTemplate;
+    }*/
+
+    @Override
+    protected void doWrite(List<? extends VariantAnnotation> variantAnnotations) {
+        //// TODO: 30/06/2016 move in constructor?
+        DBObjectToVariantAnnotationConverter converter = new DBObjectToVariantAnnotationConverter();
+
+        for (VariantAnnotation variantAnnotation : variantAnnotations) {
+            //logger.debug("Writing into mongo {}", variantAnnotation);
+
+            String storageId = buildStorageId(variantAnnotation.getChromosome(), variantAnnotation.getStart(),
+                    variantAnnotation.getReferenceAllele(), variantAnnotation.getAlternativeAllele());
+
+            BasicDBObject find = new BasicDBObject("_id", storageId);
+
+            //find existing variant with same Id, and bring only the annotation
+            DBObject existingVariantStorage = mongoOperations.getCollection(collection)
+                    .findOne(find, new BasicDBObject("annot", 1));
+
+            //update annotation in existing variant
+            if (null != existingVariantStorage){
+                if(existingVariantStorage.containsField("annot")){
+
+                    //update ConsequenceTypes
+                    VariantAnnotation existingAnnotation = converter.convertToDataModelType((DBObject) existingVariantStorage.get("annot"));
+
+                    if(null != variantAnnotation.getConsequenceTypes()){
+                        existingAnnotation.getConsequenceTypes().addAll(variantAnnotation.getConsequenceTypes());
+                    }
+
+                    //update Hgvs
+                    if(null != variantAnnotation.getHgvs()){
+                        if(null == existingAnnotation.getHgvs()){
```

Comment:
`variable != null` (or equals) instead of `null != variable` to follow the same code style than the rest.